Monika After Story/game/zz_calendar.rpy

```diff
@@ -617,13 +618,28 @@ M̼̤̱͇̤ ͈̰̬͈̭ͅw̩̜͇͈ͅa̲̩̭̩ͅs̙ ̣͔͓͚̰h̠̯̫̼͉e̗̗̮r
                         return_value=ret_val
                     )
 
+                    day_number_text = Text(
+                        self.DAY_NUMBER_DISPLAY_FORMAT.format(str(current_date.day)),
+                        font=gui.default_font,
+                        size=self.DAY_NUMBER_TEXT_SIZE,
+                        color=self.DAY_NUMBER_COLOR,
+                        outlines=[]
+                    )
+
+                    note_text = Text(
```

Comment:
Can you mark a TODO here for adding a way to decide which font to use on the calendar? 
We're thinking that standard holidays (christmas/halloween/etc...) should use the usual UI font, and only custom ones should appear handwritten. This kind of change requires rework of the calendar DB and is out of scope for this.